src/libraries/System.Utf8String.Experimental/src/System/Net/Http/Utf8StringContent.cs

```diff
@@ -40,11 +42,38 @@ public Utf8StringContent(Utf8String content, string? mediaType)
         protected override Task<Stream> CreateContentReadStreamAsync() =>
             Task.FromResult<Stream>(new Utf8StringStream(_content));
 
+#if NETSTANDARD2_0
+        protected async override Task SerializeToStreamAsync(Stream stream, TransportContext? context)
+        {
+            ReadOnlyMemory<byte> buffer = _content.AsMemoryBytes();
```

Comment:
Nit: Another opportunity for using an internal `Utf8String.GetBytes()` method which returns the `byte[]` directly. No need to bounce through `ROM<byte>`.